localizer_psychopy/localizer_exp

```diff
@@ -0,0 +1,81 @@
+# import psychopy modules
+from psychopy import visual, core, event, sound, gui, data
+
+#set parent directory
+parent_dir = "/Users/Kayleigh/fmriproject/localizer_psychopy/"
+
+# Set up text displays
+
+win = visual.Window(size = [1920,1080],
+                    color = "white",
+                    fullscr = False,
+                    units = "pix")
+
+Rhand_txt = visual.TextStim(win, text = "Curl your right hand.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+Lhand_txt = visual.TextStim(win, text = "Curl your left hand.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+Rfoot_txt = visual.TextStim(win, text = "Curl your right foot.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+Lfoot_txt = visual.TextStim(win, text = "Curl your left foot.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+tongue_txt = visual.TextStim(win, text = "Curl your tongue.",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+null_txt = visual.TextStim(win, text = "+",
+                        pos = [0,0],
+                        color = "black",
+                        height = 50,
+                        font = "Arial")
+
+
+# in the data source, there are two columns: Time (0-502) and StimType (1-6)
+# this part is trying to link the numbers in StimType to the names of the text displays
+STIM = {1: Rhand_txt, 2: Lhand_txt, 3: Rfoot_txt, 4: Lfoot_txt, 5: tongue_txt, 6: null_txt}
+# load in our stimulus timing csv file
+TRIAL_LIST = data.importConditions(fileName = parent_dir + "localizer_datasource.csv")
+
+#create clock
+globalClock = core.Clock()
+
+# start clock from 5 from scanner --- tim is going to add this part
+
+# rename clock?
+t0 = globalClock.getTime()
+
+# while the clock is running...
+while t0 >= 0:
+# Goal for this section: Check what time it is. If the time matches a value in the column "Time"
+# then present the text display from that same row for 1.25 seconds and then go back to blank
+# if  the time is greater than 503.25 seconds, end the while loop
+    for index in range(len(TRIAL_LIST)):
+        if TRIAL_LIST[index]["Time"] == t0:
```

Comment:
if you implement `t1=global.getTime()-t0` at the beginning of the while loop, here you'd change it to `if TRIAL_LIST[index]["Time"] == t1`
also, the number in `t0` and `t1` will likely have many decimal numbers, so you need to round it **down** in order to match what you read in from the file